src/polyfill.rs

```diff
@@ -17,14 +17,34 @@
 
 #![allow(unsafe_code)]
 
+use core;
+
 #[inline(always)]
 pub fn u64_from_usize(x: usize) -> u64 {
     x as u64
 }
 
+/// `core::num::Wrapping` doesn't support `rotate_left`.
+/// There is no usable trait for `rotate_left`, so this polyfill just hard-codes u32.
```

Comment:
I’m used to projects where the limit is 100 or 120, and the only help my editor gives with this is a vertical bar whose position is a global setting (not per-project).

I can add to the repo a subset of [Servo’s `tidy.py`](https://github.com/servo/servo/blob/0826a5b45455cdbecf484136b1bab39bcc0b15ff/python/tidy.py#L120-L125) to help with this.